src/foam/u2/wizard/WizardletSection.js

```diff
@@ -72,6 +72,14 @@ foam.CLASS({
         if ( ! wizardlet$of ) return true;
         if ( ! data ) return false;
 
+        if ( ! this.section ) {
+          let valid = data.validate();
+          if ( valid === undefined ) {
+            valid = ! data.errors_ || data.errors_.length < 1;
+          }
+          return valid;
```

Comment:
For validating invisible wizardlets (where the client doesn't need to enter any information?)

Reply:
This is for wizardlets with a `WizardletSection` that's not based on a model `Section`. (usually a `WizardletSection` with a `customView` set) Normally the `Section`'s error list would be used. What I've done here is make the default behaviour for this case that the entire model must validate (since wizardlets with custom views are usually just one WizardletSection anyway)